docs/web-apps/automated-testing/selenium/selenium-grid.md

```diff
@@ -0,0 +1,335 @@
+---
+id: selenium-grid
+title: Selenium Grid and Sauce Labs
+sidebar_label: Selenium Grid
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Testing new features and bug fixes is vital to enable quick software development. Selenium Grid helps with this by allowing concurrent execution of WebDriver tests across different browsers, platforms, and devices on multiple remote machines.
+
+While building your own testing infrastructure with Selenium Grid might be interesting and fun, maintaining it becomes a huge challenge over time due to the continued releases of new browser versions, browser drivers, operating systems, and mobile devices. Large amount of time needs to be invested in order to set up a fine tuned Selenium Grid that supports cross browser, cross platform, and cross device testing.
+
+By executing your tests on Sauce Labs, aside of getting a solid testing infrastructure, a wide range of features gets added on top, such as:
+- Live debugging, video recording, command tracing, screenshots, and exception highlighting to enable easy debugging on the Sauce Labs [tests results dashboard](/test-results/viewing-test-results/).
+- Sharing test results securely between team members by configuring [users and teams](/basics/acct-team-mgmt-hub/).
+- Understanding how your application renders on every device and operating system by analyzing results from [Front-End Performance Testing](/performance/).
+- Interpreting test results over time and identifying failure patterns across through [Insights](/insights/).
+- Benefitting from the integrations Sauce Labs has with all major [CI/CD platforms](/ci/).
+
+
+## What You'll Learn
+* How to extend Selenium Grid with Sauce Labs.
+* How to move from Selenium Grid to Sauce Labs.
+
+
+## What You'll Need
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* Java 11 or higher installed on every machine part of Selenium Grid.
+* Browsers and browser drivers you want to support installed on each node.
+* Selenium Server JAR file from the [latest release](https://github.com/SeleniumHQ/selenium/releases/latest).
+
+
+## How Selenium Grid Works
+Selenium is built on a [client-server architecture](/web-apps/automated-testing/selenium/#architecture), where Selenium Grid is the server. Selenium Grid has six different [components](https://www.selenium.dev/documentation/grid/components/), which gives you the option to deploy it in different ways.
```

Comment:
Suggestion to link to the Selenium Grid docs vs. duplicating the content. This will help with future maintenance + focus on the content that explains how to extend the Grid with Sauce Labs, and move away from the Grid to Sauce Labs.
```suggestion
Selenium is built on a [client-server architecture](/web-apps/automated-testing/selenium/#architecture), where Selenium Grid is the server. For details on how Selenium Grid works, its components, and how to set it up, see the [Selenium Grid Documentation](https://www.selenium.dev/documentation/grid/).
```

Reply:
In addition to the suggestion above (since it's covered in the Selenium docs), the rest of the content in this section (How Selenium Grid Works) should be removed, including the lower level sections for Standalone, Hub and Node, and Distributed.

Reply:
@diemol @bonigarcia ~ Any thoughts on this proposed change?

Reply:
I am fine with either way. This is part of the conversation we had about avoiding duplicated content vs. having all the content in our site so the reader does not leave the Sauce Labs domain.

I am not sure what is better in terms of SEO and search results in search engines. But I am happy to go with the direction you folks want to give to the content.